src/storage/signal/index.ts

```diff
@@ -0,0 +1 @@
+export * from './signal'
```

Comment:
if there is only one file per folder you can have it as index to reduce boilerplate